### ISSUE TITLE
change default gaz separator from : to tab

### ISSUE THREAD

**Issue by greenwoodma** [GateNLP/gateplugin-ANNIE#15]
it would be great if we could change the default gaz separator from : to tab. The question is what would this break? It shouldn't break any apps that just use ANNIE with the default settings as these don't get saved into the xgapp any more, but it would break any apps that have taken a copy of the ANNIE gaz files and edited them. Is it worth breaking these to give us a more sensible default separator?

**Comment by ianroberts** [comment]
Since we don't save parameter values that match the default, changing this now will break _any_ app saved in 8.5 or later with a gazetteer that either (a) deliberately uses `:` as its feature separator or (b) contains tab characters in any of its list entries.

**Comment by greenwoodma** [comment]
Yes. I'm wondering if there is a clean solution involving hiding the existing gaz PR and having a subclass with more sensible defaults, but I'm struggling to come up with a working solution.
